docs/dev/hooks.md

```diff
@@ -0,0 +1,23 @@
+# Git pre-commit hooks
+
+To minimize review overhead OpenWhisk uses tools to enforce our coding standards. This is a set of git pre-commit hooks which will automatically check for formatting errors, so you don't need to wait for a CI build to fail to find those.
```

Comment:
To minimize review overhead OpenWhisk uses tools to enforce our coding standards. ->
OpenWhisk uses tools to enforce coding standards and reduce review overhead.

Reply:
, so you don't need to wait for a CI build to fail to find those. ->
. This will save you from wasting Travis CI builds to find formatting errors.